docs/guides/typescript_project.mdx

```diff
@@ -99,7 +99,7 @@ Caveats
 ========
 
 As of version 1.0+, the generated typings, due to JSDoc limitations, have some properties
-and parameters annotated with `any` type, therefore the settings `noImplicitAny` and `strictNullChecks`, set to `true`, may not be advised. You may try enabling them, but it might hinder development because of the need for typecasts to be able to compile, your mileage may vary.
+and parameters annotated with `any` type, therefore the settings `noImplicitAny` and `strictNullChecks`, set to `true`, may not be advised. They could be enabled, but it might hinder development because of the need for typecasts to be able to compile, and thus our mileage may vary.
```

Comment:
this whole TS guide is very outdated, will try to find some time for it after i am done with the getting started guide